client/collector/src/main/java/org/hawkular/apm/client/collector/internal/DefaultTraceCollector.java

```diff
@@ -47,9 +47,9 @@
 import org.hawkular.apm.api.services.TracePublisher;
 import org.hawkular.apm.api.utils.EndpointUtil;
 import org.hawkular.apm.api.utils.PropertyUtil;
-import org.hawkular.apm.client.api.reporter.BatchTraceReporter;
 import org.hawkular.apm.client.collector.SessionManager;
 import org.hawkular.apm.client.collector.TraceCollector;
+import org.hawkular.apm.trace.publisher.rest.client.BatchTraceRecorder;
```

Comment:
This is making the collector dependent upon the REST implementation rather than the interface - I can't see a specific reason for this in the other changes to this class, was this intentional?